core/compatibility/components/PriceSelector.js

```diff
@@ -25,11 +25,16 @@ export default {
     context: {
       type: null,
       default: ''
+    },
+    checked: {
+      type: Boolean,
+      required: false,
+      default: () => false
     }
   },
   data () {
     return {
-      active: false
+      active: this.checked
```

Comment:
this should be removed. After change component should emit change and get state from `checked` prop